modules/tensor_mechanics/src/materials/GeneralizedKelvinVoigtModel.C

```diff
@@ -0,0 +1,93 @@
+/****************************************************************/
```

Comment:
Since this is the only class that derives from GeneralizedKelvinVoigtBase, does it really make sense to split this into a base and derived class? (same goes for the Maxwell model) I'm guessing that you may have done this because you have other specialized models that you want to create elsewhere for specific applications. Is that the case?

Reply:
Yes, I have done this so that we can write specialized models (notably for concrete) which could be aging, or dependent on temperature, humidity, etc. The idea is that the *Base classes contain all the math (that are defined as final methods to avoid overwriting), and the *Model classes provide the values of the parameters in all spring/dashpots.

Reply:
Ok, just checking.